tekton/src/tasks/fvt/fvt-mobile.yml.j2

```diff
@@ -0,0 +1,159 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: mas-fvt-mobile
+spec:
+  params:
+    # Control the image pull policy for the FVT container image
+    - name: image_pull_policy
+      type: string
+      default: IfNotPresent
+
+    # Test Container Information
+    # -------------------------------------------------------------------------
+    - name: fvt_image_registry
+      type: string
+      description: FVT Container Image Registry (required)
+    - name: fvt_image_digest
+      type: string
+      description: FVT Container Image Digest
+
+    # Test Framework Information
+    # -------------------------------------------------------------------------
+    - name: fvt_enable_debug
+      type: string
+      description: Turn on debug logging (verbose mode)
+      default: "true"
+    - name: fvt_mobile_app
+      type: string
+      description: The app that will be the target of the tests
+      default: ""
+    - name: fvt_test_suite
+      type: string
+      description: If the FVT container image contains multiple suites, use this to control the suite that will be executed
+      default: ""
+    - name: product_id
+      type: string
+      description: Product ID under test
+      default: ibm-mas-mobile
+    - name: product_channel
+      type: string
+      description: "Subscription channel for the product (todo: have the test code look this up instead)"
+      default: ""
+    - name: mas_instance_id
+      type: string
+      description: Instance ID of the target test environment
+    - name: mas_workspace_id
+      type: string
+      description: Workspace ID in MAS to use for running the tests
+      default: "masdev"
+
+    # Test-specific Information (all optional)
+    # -------------------------------------------------------------------------
+    - name: ddp_apikey
```

Comment:
Do you actuall use ddp_apikey, partium_username, partium_password in the test?  I thought these were only used in a handful of Core FVTs?

Reply:
We do not, but these were added a while ago in all tasks so I figured this was needed. I would be more than glad to remove these. Should I?

Reply:
I have just removed these params. Thanks!